File: library/src/main/java/com/proxerme/library/enums/Category.java
package com.proxerme.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum holding the available categories.
 *
 * @author Ruben Gees
 */
public enum Category {
    @Json(name = "anime")ANIME,
    @Json(name = "manga")MANGA
}

File: library/src/main/kotlin/me/proxer/library/enums/Category.kt
package me.proxer.library.enums

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Enum holding the available categories.
 *
 * @author Ruben Gees
 */
@JsonClass(generateAdapter = false)
enum class Category {

    @Json(name = "anime")
    ANIME,

    @Json(name = "manga")
    MANGA,

    @Json(name = "novel")
    NOVEL
}
